sdk/azurestack/Microsoft.AzureStack.Management.Storage.Admin/src/Microsoft.AzureStack.Management.Storage.Admin.csproj

```diff
@@ -16,9 +16,6 @@
                 1) Manage storage quotas 
                 2) Garbage collect deleted storage resources 
                 3) Restore deleted storage accounts 
-                4) Migrate containers from one share to another. 
```

Comment:
Why would you delete these lines but keep the above ones?

Reply:
Because we remove the 4th feature in this version by design. But the remaining 3 are kept.

Reply:
@wenjlu-lgy , only the notes of the upcoming release should be listed here. I believe the above items belong to the last release, right?

Reply:
OK, I will only note down that we remove the 4th feature.